src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/_params.py

```diff
@@ -0,0 +1,132 @@
+        c.argument('password', options_list=['--password', '-p'], help='The email password for the bot.')
+
+    with self.argument_context('bot msteams create') as c:
+        c.argument('is_disabled', options_list=['--add-disabled'], arg_type=get_three_state_flag(), help='Add the channel in a disabled state.')
+        c.argument('enable_calling', help='Enable calling on Microsoft Teams.', arg_type=get_three_state_flag())
+        c.argument('calling_web_hook', help='The calling web hook to use on Microsoft Teams.')
+
+    with self.argument_context('bot skype create') as c:
+        c.argument('is_disabled', options_list=['--add-disabled'], arg_type=get_three_state_flag(), help='Add the channel in a disabled state.')
+        c.argument('enable_messaging', help='Enable messaging on Skype.', arg_type=get_three_state_flag())
+        c.argument('enable_media_cards', help='Enable media cards on Skype.', arg_type=get_three_state_flag())
+        c.argument('enable_video', help='Enable video on Skype.', arg_type=get_three_state_flag())
+        c.argument('enable_calling', help='Enable calling on Skype.', arg_type=get_three_state_flag())
+        c.argument('enable_screen_sharing', help='Enable screen sharing on Skype.', arg_type=get_three_state_flag())
+        c.argument('enable_groups', help='Enable groups on Skype.', arg_type=get_three_state_flag())
+        c.argument('groups_mode', help='select groups mode on Skype.')
+        c.argument('calling_web_hook', help='The calling web hook to use on Skype.')
+
+    with self.argument_context('bot kik create') as c:
+        c.argument('is_disabled', options_list=['--add-disabled'], arg_type=get_three_state_flag(), help='Add the channel in a disabled state.')
+        c.argument('user_name', options_list=['--user-name', '-u'], help='kik user name.')
+        c.argument('is_validated', help='Whether or not the Kik account has been validated for use with the bot.', arg_type=get_three_state_flag())
+        c.argument('api_key', options_list=['--key'], help='the api key for the kik account.')
+
+    with self.argument_context('bot webchat create') as c:
+        c.argument('is_disabled', options_list=['--add-disabled'], arg_type=get_three_state_flag(), help='Add the channel in a disabled state.')
+        c.argument('site_name', options_list=['-s', '--site-name'], help='name of the Webchat channel site.')
+        c.argument('enable_preview', help='Enable preview features on the chat control.', arg_type=get_three_state_flag())
+
+    with self.argument_context('bot directline create') as c:
+        c.argument('is_disabled', options_list=['--add-disabled'], arg_type=get_three_state_flag(), help='Add the channel in a disabled state.')
+        c.argument('site_name', options_list=['-s', '--site-name'], help='name of the Directline channel site.')
+        c.argument('is_v1_disabled', options_list=['--disablev1'], help='Enable v1 channel protocol.', arg_type=get_three_state_flag())
```

Comment:
Argument is explicitly called `--disable`, but the help text reads `enable`. Fix.